src/lib.rs

```diff
@@ -804,3 +804,65 @@ mod serde {
         }
     }
 }
+
+#[cfg(feature = "borsh")]
+mod borsh {
```

Comment:
Let's move this into a separate module 
```suggestion
mod borsh;
```